dbadmin/migrations/021_cves_sorting_names.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE cve ALTER COLUMN name SET DATA TYPE TEXT;
+DROP COLLATION "numeric";
```

Comment:
Also in downgrade script the collation can't be dropped because it has to exist in the previous version.

Reply:
Unfortunately, downgrades are not covered by tests and we didn't have to use them in prod yet, but let's try to keep them working. 🙂